core/src/components/input/input.ionic.scss

```diff
@@ -8,9 +8,8 @@
 :host {
   --border-width: #{$ionic-border-size-small};
   --border-color: #{$ionic-color-neutral-300};
-
-  // TODO(FW-6113): Verify the ionic design token is correct once it's available and remove the hardcoded value.
-  --highlight-color-invalid: var(--ionic-color-error-600, #970606);
+  --highlight-color-invalid: #{$ionic-color-error-400};
+  --text-color-invalid: #{$ionic-color-error-600};
```

Comment:
Decided to add this new variable in order for a dev to be able to customize the text. This would follow the customization structure as md. The md theme allows a  dev to change the text color by setting `--highlight-color-invalid`. The reason why the ionic theme has separate variables is because highlight and text do not share the same colors as md is structured.